webpack/assets/javascripts/react_app/components/AutoComplete/index.js

```diff
@@ -44,10 +44,6 @@ class AutoComplete extends React.Component {
     }
     const instance = this._typeahead.current.getInstance();
     switch (e.charCode) {
-      case KEYCODES.ENTER: {
```

Comment:
@Laviro are you OK with these changes?

Reply:
Pressing `Enter` on the window, will search again the value in the input,
this means that if you are not on the first table's page,
it will bring you back to it..

Being honest.. I am not sure someone is using this functionality,
but if someone does.. maybe we should keep it.

@glekner  and I were speaking on this offline,
and maybe it is possible to add `event.stoppropagation()` to the input's keyDown handler, 
to prevent the events calling `handleSearch` twice.

Here is the behavior mentioned above:
![feb 12 2019 5_22 pm](https://user-images.githubusercontent.com/26363699/52646168-f211f980-2eea-11e9-8f98-cf7770e215d9.gif)